### PR TITLE
M3-365 Linode Power Confirmation Dialog

### PR DESCRIPTION
**Pull request by martinmckenna**
I promise this will be the last PR with unnecessary formatting changes :)

### Purpose

Adding a dialog confirmation to two destructive actions: powering down and rebooting a Linode

### CODE REVIEW

**Comment by patthiel** [comment]
Page object methods for `powerOff(linode)` and `reboot(linode)` will have to be updated in `e2e/pageobjects/list-linodes.js`

**Comment by martinmckenna** [comment]
@patthiel all ready for you

File: src/features/linodes/LinodesLanding/LinodesLanding.tsx [hunk 1]
```diff
@@ -400,6 +434,38 @@ export class ListLinodes extends React.Component<CombinedProps, State> {
...
+            <ActionsPanel style={{ padding: 0 }}>
+              <Button
+                variant="raised"
+                color="secondary"
+                className="destructive"
+                onClick={this.rebootOrPowerLinode}
+                data-qa-confirm-cancel
+              >
+                {(bootOption === 'reboot')
+                  ? 'Reboot Linode'
```
Comment (rjstires) [inline src/features/linodes/LinodesLanding/LinodesLanding.tsx]: The word "Linode" is superfluous in the button and can be removed.

**Comment by patthiel** [comment]
Tests updated, you are good to go @martinmckenna  🌮 👍

**Pull request merged**